admin/app/views/readerRevenue/subscriptionsBannerDeploys.scala.html

```diff
@@ -0,0 +1,31 @@
+@import common.LinkTo
+@import conf.Configuration
+@import model.readerRevenue._
+@import model.readerRevenue.ReaderRevenueRegion
+@(regions: List[ReaderRevenueRegion])(implicit request: RequestHeader, context: model.ApplicationContext, flash: Flash)
+
+
+@redeployButton(region: ReaderRevenueRegion) = {
+    <form action="@controllers.admin.routes.ReaderRevenueAdminController.redeploySubscriptionsBanner(region.name)" method="POST">
+        <input class="btn btn-large btn-danger" type="submit" value="Redeploy banner in @region.name">
+    </form>
+}
+
+@admin_main("Redeploy Contributions Banner", isAuthed = true) {
+
+    <h1>Subscriptions Banner Redeploy</h1>
+
+    <p>Click buttons to redeploy subscriptions banner</p>
+
+    <ul>
+        @for(region <- regions) {
+            <li>
+                @redeployButton(region)
+                <p><a href="@Configuration.ajax.url/reader-revenue/subscriptions-banner-deploy-log/@region.name">Timestamp of last banner redeploy</a></p>
```

Comment:
I'm not sure this should be on the ajax URL technically?

Reply:
this used to see the last deploy time for each environment, so it is needed